network-api/networkapi/mozfest/templates/mozfest/mozfest-base.html

```diff
@@ -8,9 +8,8 @@
 {% endblock %}
 
 {% block social_metadata %}
-    <meta property='og:type' content='website'>
-    <meta property='og:locale' content='en_US'>
-    {% meta_tags %}
+  <meta property='og:type' content='website'>
+  <meta property='og:locale' content='en_US'>
```

Comment:
mozfest is simply en_US

Reply:
Why is that if you don't mind me asking? 😬

Reply:
Yeah, there are localization fields for a bunch of the content on it...why hardcode it to English?

Reply:
Also, was this just a spacing fix?

Reply:
It was, although if mozfest is properly localized this needs to simply "not override the social_metadata"